Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -14,6 +14,8 @@
 		806D559125AFD735005ED326 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 806D559025AFD735005ED326 /* Assets.xcassets */; };
 		806D559725AFD735005ED326 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 806D559625AFD735005ED326 /* main.m */; };
 		806D55B725AFD877005ED326 /* PopupBridge_DemoUITests.m in Sources */ = {isa = PBXBuildFile; fileRef = 806D55B625AFD877005ED326 /* PopupBridge_DemoUITests.m */; };
+		806F556025B1400A001754AF /* PopupBridge.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 806F555F25B1400A001754AF /* PopupBridge.framework */; };
+		806F556125B1400A001754AF /* PopupBridge.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 806F555F25B1400A001754AF /* PopupBridge.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
Relink & embed PopupBridge framework to Demo app (must have forgotten to commit in previous PR ? )